setup.py

```diff
@@ -16,5 +16,6 @@
     install_requires=[
         'demands >= 1.0.5, < 2.0.0',
         'property-caching >= 1.0.0, < 2.0.0',
+        'six >= 1.4.0, < 2.0.0',
```

Comment:
1.4.1 looks like it fixes a bug that may hit us: https://bitbucket.org/gutworth/six/issues/32
Ay particular reason for going with 1.4.0?